ui/app/templates/components/job-editor/edit.hbs

```diff
@@ -13,17 +13,32 @@
     {{/if}}
     </div>
     <div class="boxed-section-body is-full-bleed">
-    <div
-        data-test-editor
-        {{code-mirror
-        screenReaderLabel="Job definition"
-        content=@data.job._newDefinition
-        theme="hashi"
-        onUpdate=@fns.onUpdate
-        mode="javascript"
-        }}
-    />
+        <div
+            data-test-editor
+            {{code-mirror
+            screenReaderLabel="Job definition"
+            content=@data.job._newDefinition
+            theme="hashi"
+            onUpdate=@fns.onUpdate
+            mode="javascript"
+            }}
+        />
+    </div>
+    {{#if (eq @data.view "job-spec")}}
+    <div>
+        <h1 style="font-size: 2rem; font-weight: 600">Edit HCL Variables</h1>
+        <div
+            data-test-variable-editor
+            {{code-mirror
+            screenReaderLabel="HLC Variables for Job Spec"
+            content=""
```

Comment:
This is a brilliant comment. No, it won't. This will instead, take us into `review` stage of the component on success. Unless, there is a `parse` failure. As of now, we do not have a variable parser and it won't be in scope for the project, so this may always start as blank.

It may benefit us to create a collection of API requests/responses to inform us about the nuances of the user experience here.